{"status": "broken", "description": "Activating the pick action test and checking whether it was successful.\n        ", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "complex_test"}, {"name": "subSuite", "value": "TestComplexScenario"}, {"name": "host", "value": "Hyper-Dock"}, {"name": "thread", "value": "18321-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython2"}, {"name": "package", "value": "tests.complex_test"}], "stop": 1608371973410, "uuid": "bf986b1c-cd19-41c0-8d25-9d7ad3a23b51", "historyId": "6df5cbf7c00ce5f14b4c42d69bcd2de9", "name": "test_verification_of_pick_action", "start": 1608371971292, "fullName": "tests.complex_test.TestComplexScenario#test_verification_of_pick_action", "statusDetails": {"message": "TypeError: __init__() takes exactly 2 arguments (1 given)", "trace": "self = <tests.complex_test.TestComplexScenario instance at 0x7f621a93ed88>\n\n    def test_verification_of_pick_action(self):\n        \"\"\"Activating the pick action test and checking whether it was successful.\n        \"\"\"\n        data_logger('logger/logs/pick_action_start')\n        # Pick from table and place on table\n        result = MoveItPickAndPlace(pick_x = -0.03, pick_y = 0.96 , pick_z = 0.57,\n>                                   place_x = 0.5, place_y = 1.0 , place_z = 0.6)\n\ntests/complex_test.py:173: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\npick_x = -0.03, pick_y = 0.96, pick_z = 0.57, place_x = 0.5, place_y = 1.0\nplace_z = 0.6\n\n    def MoveItPickAndPlace(pick_x = 0, pick_y = 0, pick_z = 0, place_x = 0, place_y = 0, place_z = 0):\n        # Prepare hsrb_interface\n        robot = hsrb_interface.Robot()\n        whole_body = robot.get(\"whole_body\")\n        omni_base = robot.get(\"omni_base\")\n        gripper = robot.get(\"gripper\")\n>       collision_world = hsrb_interface.collision_world.CollisionWorld()\nE       TypeError: __init__() takes exactly 2 arguments (1 given)\n\ntests/action_client/experimental_pick_place_client.py:27: TypeError"}, "attachments": [{"source": "5944d66a-10af-4aac-9793-9c9fe6a22179-attachment.txt", "type": "text/plain", "name": "log"}, {"source": "d598697f-a4d1-4cc3-8f62-7ab94c5200f5-attachment.txt", "type": "text/plain", "name": "stdout"}, {"source": "5df42ec1-c696-4838-850f-44840de5ab61-attachment.txt", "type": "text/plain", "name": "stderr"}]}